projects/components/src/datagrid/filters/datagrid-numeric-filter.component.spec.ts

```diff
@@ -53,24 +98,52 @@ describe('Datagrid numeric filter', () => {
-            this.filter.setValue([1, null]);
-            expect(this.filter.getValue()).toEqual(`${queryFieldName}=gt=1`);
-        });
-        it('returns a FIQL string with both ge and le as operators when both limits are set', function(this: HasDgNumericFilter): void {
-            this.filter.setValue([1, 10]);
-            expect(this.filter.getValue()).toEqual(`(${queryFieldName}=ge=1;${queryFieldName}=le=10)`);
-        });
-        it('returns a FIQL string with lt as operator when only to is set', function(this: HasDgNumericFilter): void {
-            this.filter.setValue([null, 10]);
-            expect(this.filter.getValue()).toEqual(`${queryFieldName}=lt=10`);
-        });
-        it('returns a FIQL string with both ge and le as operators when both limits are 0', function(this: HasDgNumericFilter): void {
-            this.filter.setValue([0, 0]);
-            expect(this.filter.getValue()).toEqual(`(${queryFieldName}=ge=0;${queryFieldName}=le=0)`);
+        describe('without units', () => {
+            beforeEach(function(this: HasDgNumericFilter): void {
+                this.filter = createDatagridFilterTestHelper(DatagridNumericFilterComponent);
+            });
+            it('returns a FIQL string with gt as operator when only from is set', function(this: HasDgNumericFilter): void {
+                this.filter.setValue([1, null]);
+                expect(this.filter.getValue()).toEqual(`${queryFieldName}=gt=1`);
+            });
+            it('returns a FIQL string with both ge and le as operators when both limits are set', function(this: HasDgNumericFilter): void {
+                this.filter.setValue([1, 10]);
+                expect(this.filter.getValue()).toEqual(`(${queryFieldName}=ge=1;${queryFieldName}=le=10)`);
+            });
+            it('returns a FIQL string with lt as operator when only to is set', function(this: HasDgNumericFilter): void {
+                this.filter.setValue([null, 10]);
+                expect(this.filter.getValue()).toEqual(`${queryFieldName}=lt=10`);
+            });
+            it('returns a FIQL string with both ge and le as operators when both limits are 0', function(this: HasDgNumericFilter): void {
+                this.filter.setValue([0, 0]);
+                expect(this.filter.getValue()).toEqual(`(${queryFieldName}=ge=0;${queryFieldName}=le=0)`);
+            });
+        });
+        describe('with units', () => {
+            beforeEach(function(this: HasFinderAndFilter): void {
+                const finderAndFilter = createDatagridFilterTestHelperWithFinder(DatagridNumericFilterComponent, {
+                    unitOptions: [...Bytes.types],
+                    unit: Bytes.MB,
+                });
+                this.finder = finderAndFilter.finder;
+                this.filter = finderAndFilter.filter;
+            });
+            // tslint:disable-next-line:max-line-length
```

Comment:
Please avoid this `// tslint:disable-next-line:max-line-length`, break up your string if necessary